packages/linter/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Not sure this is needed as there's one in the repo root.

Reply:
Yes, you're right looks like a LICENSE is somehow added to the packages: https://unpkg.com/browse/@ampproject/toolbox-optimizer@1.1.1/LICENSE. Maybe lerna does this? `npm pack` (which I thought lerna would be using) doesn't.